.gitignore

```diff
@@ -1,2 +1 @@
-test.md
 node_modules
```

Comment:
- [ ] Kindly add your test.md file to your .gitignore file. It is a great way to practice how to use .gitignore file 😁😁